frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -77,9 +82,84 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool cleanup -p {{TASKCFG_ALL_JMX_PORT}} -- $CASSANDRA_KEYSPACE_TABLES
         resource-set: sidecar-resources
+      backup-schema:
+        goal: FINISHED
+        cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/cqlsh -e "DESC SCHEMA" "node-${POD_INSTANCE_INDEX}-server.{{SERVICE_NAME}}.mesos" > container-path/schema.cql
+        resource-set: sidecar-resources
       snapshot:
         goal: FINISHED
-        cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool snapshot -p {{TASKCFG_ALL_JMX_PORT}} -t $CASSANDRA_SNAPSHOT_NAME -- $CASSANDRA_KEYSPACES
+        cmd: >
```

Comment:
For the large `cmd`s here and below, I think it might be a bit nicer to separate these out into separate files.